buildkite/src/Command/MinaArtifact.dhall

```diff
@@ -262,27 +288,48 @@ let docker_commands
                 )
                 flattened_docker_steps
 
+let pipelineBuilder
+    : MinaBuildSpec.Type -> List Command.Type -> Pipeline.Config.Type
+    =     \(spec : MinaBuildSpec.Type)
+      ->  \(steps : List Command.Type)
+      ->  Pipeline.Config::{
+          , spec = JobSpec::{
+            , dirtyWhen = DebianVersions.dirtyWhen spec.debVersion
+            , path = "Release"
+            , name =
+                "${spec.prefix}${DebianVersions.capitalName
```

Comment:
Isn't `-` delimeter needed between parts?

Reply:
No, becasue it is added in toSuffixUppercase methid